### PR TITLE
Added integrated security to main database runtime settings

### PR DESCRIPTION
**Pull request by grootjans**

### CODE REVIEW

**Comment by grootjans** [comment]
Please let someone from Data storage approve/review!

File: content/en/docs/refguide/runtime/custom-settings/_index.md [hunk 1]
```diff
@@ -98,6 +98,7 @@ The settings below influence the behavior of the log files. These settings can o
 | **DatabaseName** | The name of the database or schema used by the Mendix app <br/>This will be overridden if you supply **DatabaseJdbcUrl**. | |
 | **DatabaseJdbcUrl** | Defines the JDBC URL to use for the database connection (which overrides the other database connection settings). |   |
 | **DatabaseUseSsl** | For PostgreSQL databases, defines whether the connection will be made using SSL without certificate validation. If you need certificate validation, use **DatabaseJdbcUrl** instead. | false |
+| **DatabaseUseIntegratedSecurity** | This setting defines whether integrated security should be used for SQL Server. If true, user name and password will not be used. | false |
```
Comment (basdebakker) [inline content/en/docs/refguide/runtime/custom-settings/_index.md]: Maybe make "used for" slightly more explicit? Like "used to connect to" or "used to login to".

Comment (grootjans) [reply content/en/docs/refguide/runtime/custom-settings/_index.md]: Updated, and also updated the sourcedatabase variant

**Review by basdebakker — APPROVED** [review]
Approved by datastorage

File: content/en/docs/refguide/runtime/custom-settings/_index.md [hunk 2]
```diff
@@ -98,6 +98,7 @@ The settings below influence the behavior of the log files. These settings can o
 | **DatabaseName** | The name of the database or schema used by the Mendix app <br/>This will be overridden if you supply **DatabaseJdbcUrl**. | |
 | **DatabaseJdbcUrl** | Defines the JDBC URL to use for the database connection (which overrides the other database connection settings). |   |
 | **DatabaseUseSsl** | For PostgreSQL databases, defines whether the connection will be made using SSL without certificate validation. If you need certificate validation, use **DatabaseJdbcUrl** instead. | false |
+| **DatabaseUseIntegratedSecurity** | This setting defines whether integrated security will be used to authenticate to SQL Server. If true, user name and password will not be used. | false |
```
Comment (MarkvanMents) [inline content/en/docs/refguide/runtime/custom-settings/_index.md:101]: Is this for all Mx 9.x.x or was it introduced later?

File: content/en/docs/refguide7/runtime/custom-settings/_index.md [hunk 3]
```diff
@@ -70,6 +70,7 @@ The settings below influence the behavior of the log files. These settings can o
 | ClientQueryTimeout | Defines the timeout in seconds for most of the database queries which are executed to load data into client widgets, like data grids. After the duration as specified here, a query will be canceled and an exception will be thrown. |   |
 | DatabaseJdbcUrl | Defines the JDBC URL to use for the database connection (which overrides the other database connection settings). This feature is not supported for PostgreSQL databases. |   |
 | DatabaseUseSsl | For PostgreSQL databases, defines whether the connection will be made using SSL. | false |
+| DatabaseUseIntegratedSecurity | This setting defines whether integrated security will be used to authenticate to SQL Server. If true, user name and password will not be used. | false |
```
Comment (MarkvanMents) [inline content/en/docs/refguide7/runtime/custom-settings/_index.md:73]: Is this for all Mendix 7 versions, or just the latest LTS? If so, when was it introduced?

File: content/en/docs/refguide8/runtime/custom-settings/_index.md [hunk 4]
```diff
@@ -78,6 +78,7 @@ The settings below influence the behavior of the log files. These settings can o
 | **DatabaseName** | The name of the database or schema used by the Mendix app <br/>This will be overridden if you supply **DatabaseJdbcUrl**. | |
 | **DatabaseJdbcUrl** | Defines the JDBC URL to use for the database connection (which overrides the other database connection settings). |   |
 | **DatabaseUseSsl** | For PostgreSQL databases, defines whether the connection will be made using SSL without certificate validation. If you need certificate validation, use **DatabaseJdbcUrl** instead. | false |
+| **DatabaseUseIntegratedSecurity** | This setting defines whether integrated security will be used to authenticate to SQL Server. If true, user name and password will not be used. | false |
```
Comment (MarkvanMents) [inline content/en/docs/refguide8/runtime/custom-settings/_index.md:81]: Is this for all Mendix 8 versions, or just the latest LTS? If so, when was it introduced?

**Review by MarkvanMents — COMMENTED** [review]
Hi @grootjans and @basdebakker 
I can't find anything in Slack or the PR description to identify if this is an omission since 7.0.0 or whether it has been introduced recently.
Can you confirm that the new setting applies to all versions of Mendix from 7.0.0 upwards, or let me know when it was introduced.
Thanks.
Otherwise, ready to be merged.
Thanks for providing this PR.
Mark

File: content/en/docs/refguide7/runtime/custom-settings/_index.md (hunk 3 above)
Comment (basdebakker) [reply content/en/docs/refguide7/runtime/custom-settings/_index.md:73]: This was just an omission in the documentation. I don't know when exactly the setting was introduced, but it has been there in all version of Mendix from 7.0 onwards.

Comment (MarkvanMents) [reply content/en/docs/refguide7/runtime/custom-settings/_index.md:73]: Great - I'll publish now.

**Review by MarkvanMents — APPROVED** [review]
Thanks for the update. Approving and publishing.

**Pull request merged**